css/properties/-moz-user-focus.json

```
@@ -11,7 +11,8 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "version_removed": "123"
```

According to https://bugzilla.mozilla.org/show_bug.cgi?id=1871745

```suggestion
              "version_removed": "122"
```